src/app/store/localStorageMiddleware.ts

```diff
@@ -25,6 +24,7 @@ import {
     LSKEY_APPLICATION_MEDIA_SEARCH_WALL_VIEW_MODE,
     LSKEY_APPLICATION_THEME,
     LSKEY_APPLICATION_USE_IMAGE_BACKGROUND,
+    LSKEY_APPLICATION_VOLUME_LIMIT,
```

Comment:
Congratulations on successfully navigating the local storage middleware system! 😄  It's pretty heinous.

And I like how you've rebranded "max volume" as "volume limit". Much more accurate now.